docs/release_notes/v1.3.1.adoc

```diff
@@ -0,0 +1,44 @@
+[id="{p}-release-notes-v1.3.1"]
+=== Elastic Cloud Control (ecctl) v1.3.1
+++++
+<titleabbrev>v1.3.1</titleabbrev>
+++++
+
+<<{p}-release-notes-v1.3.1-bug-fixes,Bug fixes>> | <<{p}-release-notes-v1.3.1-changelog,Changelog>>
+
+Welcome to the v1.3.1 release of {n}. This version brings a bug fix.
+
+Elastic Cloud Control (ecctl) is Elastic’s CLI to manage Elasticsearch Service (ESS) and Elastic Cloud Enterprise (ECE).
+
+Download the release binaries:
+
+[%hardbreaks]
+https://download.elastic.co/downloads/ecctl/1.3.1/ecctl_1.3.1_darwin_amd64.tar.gz[ecctl_1.3.1_darwin_amd64.tar.gz]
+https://download.elastic.co/downloads/ecctl/1.3.1/ecctl_1.3.1_linux_32-bit.deb[ecctl_1.3.1_linux_32-bit.deb]
+https://download.elastic.co/downloads/ecctl/1.3.1/ecctl_1.3.1_linux_32-bit.rpm[ecctl_1.3.1_linux_32-bit.rpm]
+https://download.elastic.co/downloads/ecctl/1.3.1/ecctl_1.3.1_linux_386.tar.g[ecctl_1.3.1_linux_386.tar.g]
```

Comment:
Is this suppose to be `.gz`?

Reply:
Yeah it is. The template must be broken then